tests/multigrid-global-coarsening/multigrid_p_01.cc

```diff
@@ -129,8 +129,8 @@ test(const unsigned int n_refinements,
                                                 constraints[l]);
 
   MGTransferGlobalCoarsening<dim, VectorType> transfer(
-    transfers,
-    [&](const auto l, auto &vec) { operators[l].initialize_dof_vector(vec); });
+    transfers/*,
+    [&](const auto l, auto &vec) { operators[l].initialize_dof_vector(vec); }*/);
```

Comment:
Can you explain this change? Doesn't this mean that you change the code to trigger the assertion (or why isn't the call to `interpolate_to_mg` triggering it)?

Reply:
> Doesn't this mean that you change the code to trigger the assertion (or why isn't the call to interpolate_to_mg triggering it)?

Yes. I wanted to test it locally. It shouldn't have got into the PR. I have removed it!